test/integration_test/migration_test.go

```diff
@@ -2027,6 +2032,288 @@ func transformResourceTest(t *testing.T) {
+		[]string{instanceID},
+		false,
+		false,
+		true,
+		false,
+		"",
+		nil)
+
+	err = schedulerDriver.WaitForRunning(ctxs[0], defaultWaitTimeout/2, defaultWaitInterval)
+	require.NoError(t, err, "Migration could not be completed")
+
+	// Change kubeconfig to destination
+	err = setDestinationKubeConfig()
+	require.NoError(t, err, "failed to set kubeconfig to source cluster: %v", err)
+
+	destDeployments, err := apps.Instance().ListDeployments(namespace, meta_v1.ListOptions{})
+	require.NoError(t, err, "error retriving deployments list from %s namespace", namespace)
+	require.Equal(t, 0, len(destDeployments.Items), fmt.Sprintf("Expected no deployments in destination in %s namespace", namespace))
+
+	destroyAndWait(t, []*scheduler.Context{preMigrationCtx})
+
+	err = setSourceKubeConfig()
+	require.NoError(t, err, "failed to set kubeconfig to source cluster: %v", err)
+
+	destroyAndWait(t, ctxs)
+
+	blowNamespacesForTest(t, instanceID, appKey, false)
+	// If we are here then the test has passed
+	testResult = testResultPass
+	logrus.Infof("Test status at end of %s test: %s", t.Name(), testResult)
+}
+
+func excludeResourceTypePVCTest(t *testing.T) {
+	var testrailID, testResult = 93403, testResultFail
+	runID := testrailSetupForTest(testrailID, &testResult)
+	defer updateTestRail(&testResult, testrailID, runID)
+	instanceID := "exclude-resourcetype-pvc"
+	appKey := "mysql-1-pvc"
+	pvcName := "mysql-data"
+	namespace := fmt.Sprintf("%s-%s", appKey, instanceID)
+
+	var err error
+	defer func() {
+		err = setSourceKubeConfig()
+		require.NoError(t, err, "Error resetting source config")
+	}()
+
+	ctxs, preMigrationCtx := triggerMigration(
+		t,
+		instanceID,
+		appKey,
+		[]string{},
+		[]string{instanceID},
+		false,
+		false,
+		true,
+		false,
+		"",
+		nil)
+
+	err = schedulerDriver.WaitForRunning(ctxs[0], defaultWaitTimeout/2, defaultWaitInterval)
+	require.NoError(t, err, "Migration could not be completed")
+
+	// Change kubeconfig to destination
+	err = setDestinationKubeConfig()
+	require.NoError(t, err, "failed to set kubeconfig to source cluster: %v", err)
+
+	_, err = core.Instance().GetPersistentVolumeClaim(pvcName, namespace)
+	require.Error(t, err, "getting pvc in destination in %s namespace should have failed", namespace)
+
+	destroyAndWait(t, []*scheduler.Context{preMigrationCtx})
+
+	err = setSourceKubeConfig()
+	require.NoError(t, err, "failed to set kubeconfig to source cluster: %v", err)
+
+	destroyAndWait(t, ctxs)
+
+	blowNamespacesForTest(t, instanceID, appKey, false)
+	// If we are here then the test has passed
+	testResult = testResultPass
+	logrus.Infof("Test status at end of %s test: %s", t.Name(), testResult)
+}
+
+func excludeMultipleResourceTypesTest(t *testing.T) {
+	var testrailID, testResult = 93404, testResultFail
+	runID := testrailSetupForTest(testrailID, &testResult)
+	defer updateTestRail(&testResult, testrailID, runID)
+	instanceID := "exclude-multiple-resourcetypes-migration"
+	appKey := "mysql-1-pvc"
```

Comment:
I can see that mysql-1-pvc will schedule a pvc,deployment and service in the source cluster
So we should also validate that PVC was migrated in this case

Reply:
Same thing in other places?
The resource type which is not excluded we should validate was successfully migrated?

Reply:
done